docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -191,6 +191,18 @@ To filter column values, click the filter (funnel) icon next to a column title.
 
 Click the check box next to the values that you want to display. Enter text in the search field at the top to show those values in the display so that you can select them rather than scroll to find them.
 
+### Select column values
+
+To select the subset of column values currently displayed.
+
+Click the **Select values** button to add the current subset of visible values to the full set of filter column values.
```

Comment:
I'm trying to simplify the language a bit here. Hopefully I haven't changed the meaning.
```suggestion
Select all displayed column values.

Click the **Select values** button to add all displayed values to the filter.
```